resources/views/hardware/view.blade.php

```diff
@@ -887,11 +882,13 @@
                                 @endif
 
                                 @if (($asset->assignedTo) && ($asset->deleted_at==''))
-                                    <h2>{{ trans('admin/hardware/form.checkedout_to') }}</h2>
+                                    <div style="text-align: left">
+                                        <h2>{{ trans('admin/hardware/form.checkedout_to') }}</h2>
                                         <p>
                                         @if($asset->checkedOutToUser()) <!-- Only users have avatars currently-->
-                                            <img src="{{ $asset->assignedTo->present()->gravatar() }}" class="user-image-inline" alt="{{ $asset->assignedTo->present()->fullName() }}">
```

Comment:
Why was this class removed?

Reply:
Oh that might have been accidentally removed when I was getting rid of the image I put in to test originally and just forgot that was supposed to be there.

Reply:
this might have been an accident when clearing out something else